pkg/osquery/tables/dataflattentable/exec.go

```diff
@@ -29,18 +28,13 @@ func WithKVSeparator(separator string) ExecTableOpt {
 	}
 }
 
-func WithBinDirs(binDirs ...string) ExecTableOpt {
-	return func(t *Table) {
-		t.binDirs = binDirs
-	}
-}
-
-func TablePluginExec(logger log.Logger, tableName string, dataSourceType DataSourceType, execArgs []string, opts ...ExecTableOpt) *table.Plugin {
+func TablePluginExec(logger log.Logger, tableName string, dataSourceType DataSourceType, cmd allowedcmd.AllowedCommand, execArgs []string, opts ...ExecTableOpt) *table.Plugin {
 	columns := Columns()
 
 	t := &Table{
 		logger:            level.NewFilter(logger, level.AllowInfo()),
 		tableName:         tableName,
+		cmd:               cmd,
```

Comment:
```suggestion
		cmdGen:               cmdGen,
```

Not really important, but this isn't a cd. It's a generator.